packages/unlock-js/scripts/compressAbi.js

```diff
@@ -11,7 +11,17 @@ const fs = require('fs')
 const path = require('path')
 const abis = require('@unlock-protocol/contracts')
 
-const unlockVersions = ['v4', 'v6', 'v7', 'v8', 'v9', 'v10', 'v11', 'v12']
+const unlockVersions = [
+  'v4',
+  'v6',
+  'v7',
+  'v8',
+  'v9',
+  'v10',
+  'v11',
+  'v12',
+  'v13',
+]
 const publicLockVersions = [
```

Comment:
same here. I'd sayd anything that has not seen any activity in 2 years can be removed

Reply:
yes will do in another PR